build/stf-run-ci/tasks/setup_stf_from_bundles.yml

```diff
@@ -90,10 +90,14 @@
 - name: "Ensure that the bundle paths are set."
   ansible.builtin.assert:
     that:
-      - '__smart_gateway_bundle_image_path is defined and __smart_gateway_bundle_image_path != None'
-      - '__service_telemetry_bundle_image_path is defined and __service_telemetry_bundle_image_path != None'
+      - '__smart_gateway_bundle_image_path is defined'
```

Comment:
Instead of these three checks per variable, I think the idiomatic ansible thing is this:
```
'__smart_gateway_bundle_image_path | default("") | length > 0
```

Reply:
That's fair.

If the var is undefined, it becomes "", and the check fails, because of length
If the var is None, there's an error because None does not have a length
If the var is an empty string, the check fails because of the length

Okay, I'm convinced. And there was a job failure so the CI would run again anyway

Reply:
I'm leaving the success message as-is